src/main/java/sc/fiji/pQCT/selectroi/RoiSelector.java

```diff
@@ -1084,16 +1084,18 @@ else if (result[i + j * width] != 1) {
 		return sleeve;
 	}
 
-	void erode(final byte[] data) {
+	public byte[] erode(byte[] data) {
 		// Erode algorithm
 		// Modified from the best dilate by one solution taken from
 		// http://ostermiller.org/dilate_and_erode.html
-		for (int i = 0; i < height; i++) {
-			for (int j = 0; j < width; j++) {
+		for (int i = 1; i < height-1; i++) {
+			for (int j = 1; j < width-1; j++) {
 				if (data[i * width + j] > 0) {
-					if (i > 0 && data[(i - 1) * width + j] == 0 || j > 0 && data[(i) *
-						width + j - 1] == 0 || i + 1 < height && data[(i + 1) * width +
-							j] == 0 || j + 1 < width && data[(i) * width + j + 1] == 0)
+					if (	data[(i - 1)	* width + j] == 0
+						| 	data[(i) 		* width + j - 1] == 0
+						| 	data[(i + 1)	* width +j] == 0
+						| 	data[(i)		* width + j + 1] == 0
+						)
```

Comment:
Potential bug: `|` != `||`

Reply:
This is akin to the previous similar comment. Again I want the if to evaluate to true if any of the ... == 0 is true -> should be the same result independent of whether it is | or ||.